src/app/components/chat/window/send/form/form.vue

```diff
@@ -1,5 +1,6 @@
 <script lang="ts">
```

Comment:
we did a lot of work in this file. should we bite the bullet and just convert it over to a new script/setup form?